### PR TITLE
Adds support for FTRIM (flat trim) command.

### PR DESCRIPTION
**Pull request by michaelshmitty**
See https://github.com/felixge/node-ar-drone/blob/master/lib/Client.js#L214

Signed-off-by: Michael Smith <m@michaelsmith.be>

### CODE REVIEW

**Comment by michaelshmitty** [comment]
Though it seems to work, I'm not sure I've implemented this correctly.
I'm confused because there seem to be two files where commands are defined:
https://github.com/hybridgroup/cylon-ardrone/blob/master/lib/commands.js
and
https://github.com/hybridgroup/cylon-ardrone/blob/master/lib/flight.js

What's the difference? I don't have a ton of experience with nodejs.

Great work on Cylonjs by the way. Thanks!

File: lib/flight.js [hunk 1]
```diff
@@ -124,3 +124,13 @@ Flight.prototype.rightFlip = function() {
...
   return this.connection.animate("wave", 750);
 };
+
+/**
+ * Tells the drone to perform horizontal trimming.
+ *
+ * @return {null}
+ * @publish
+ */
+Flight.prototype.ftrim = function() {
```
Comment (michaelshmitty) [inline lib/flight.js]: Not sure this is actually necessary? Is it sufficient to add the command to commands.js?

Comment (deadprogram) [inline lib/flight.js]: You are correct, just adding it to `commands.js` will cause `ftrim` to be proxied to the underlying ardrone connection https://github.com/hybridgroup/cylon-ardrone/blob/master/lib/ardrone.js#L114

Comment (michaelshmitty) [inline lib/flight.js]: @deadprogram cool, thanks Ron, I'll update the code and the pull request as soon as I can.

Out of curiosity, why are some methods explicitly (re)defined in the flight.js file?

Comment (deadprogram) [inline lib/flight.js]: The only reason they should be, is if they have a different API signature, or are a higher-level abstraction. Anything that is a straight proxy does not need to be in there.

**Comment by michaelshmitty** [comment]
Got it, thanks!

File: lib/flight.js (hunk 1 above)
Comment (michaelshmitty) [inline lib/flight.js]: Got it, thanks!

**Comment by deadprogram** [comment]
:shipit:

**Pull request merged**